string source        # who sent this message
float32 linear_speed
float32 angular_speed
float32 acceleration
float32 deceleration
bool    enable_motion
